app/models/super_good/solidus_taxjar/settings.rb

```diff
@@ -0,0 +1,6 @@
+module SuperGood
+  module SolidusTaxjar
+    class Settings < ActiveRecord::Base
```

Comment:
We had a conversation IRL about maybe using Solidus's built-in preferences stores to keep TaxJar extension settings in the DB. Has any more thought been put into that since our conversation?

(I'm guessing no. Everyone's been pretty busy.)